src/tool/subcommands/api_cmd.rs

```diff
@@ -606,6 +607,13 @@ fn snapshot_tests(store: Arc<ManyCar>, n_tipsets: usize) -> anyhow::Result<Vec<R
         tests.push(RpcTest::identity(
             ApiInfo::chain_get_messages_in_tipset_req(tipset.key().clone()),
         ));
+        tests.push(RpcTest::identity(
+            ApiInfo::state_deal_provider_collateral_bounds_req(
+                PaddedPieceSize(1),
```

Comment:
Is a single `RpcTest` enough to cover all cases?

Reply:
It's a snapshot test. It's enough.